delocate/tools.py

```diff
@@ -246,18 +433,27 @@ def get_install_names(filename):
     install_names : tuple
         tuple of install names for library `filename`
     """
-    lines = _cmd_out_err(["otool", "-L", filename])
-    if not _line0_says_object(lines[0], filename):
+    otool = subprocess.run(
```

Comment:
Maybe add `Raises` section in docstring for not-matching architectures?

Reply:
I might need to change the exceptions to tell the difference between a `_check_ignore_archs` failure and other issues.

Reply:
I've added `Raises` sections to multiple functions.  I'm now using `NotImplementedError` to indicate that a new function that can handle multiple multiple architectures needs to be added eventually.